src/openms/include/OpenMS/ANALYSIS/ID/IDScoreSwitcherAlgorithm.h

```diff
@@ -243,13 +294,15 @@ namespace OpenMS
 
     /// will be set according to the algorithm parameters
     String new_score_, new_score_type_, old_score_;
+
     /// will be set according to the algorithm parameters
     bool higher_better_; // for the new scores, are higher ones better?
 
     /// a map from ScoreType to their names as used around OpenMS
     std::map<ScoreType, std::set<String>> type_to_str_ =
         {
-            {ScoreType::RAW, {"XTandem", "OMSSA", "SEQUEST:xcorr", "Mascot", "mvh", "Sage"}},
+            //TODO introduce real meaningful score names for XTandem, Mascot etc. (e.g., hyperscore)
+            {ScoreType::RAW, {"svm", "MS:1001492", "XTandem", "OMSSA", "SEQUEST:xcorr", "Mascot", "mvh", "hyperscore", "ln(hyperscore)"}},
```

Comment:
same here: double check if hyperscore / ln(hyperscore) distinciton makes sense

Reply:
Well it would be very helpful because then you know the actual domain you are working with.

Reply:
in case you want to make a finer distinction in raw types

Reply:
I checked. Both X!Tandem and Sage use ln. For consistency reasons we could just use "hyperscore"